index.html

```diff
@@ -28,6 +28,87 @@ <h1 class=" box-item headline"> Hey There.I'm Ahmed <br> <span id="headline-gree
         </ul>
     </section>
     </main>
+    <section class="works">
+        <div class="action-btn"><img src="img/Disabled.png" alt=""></div>
+        <h1 class="headline"> My Recent Works </h1>   
+        <div class="main-container">
+            <div class="grid-item"> 
+                <div class="flex-item1"><img src="img/Plaholder.png" alt="placeholder"></div>
+                <div class="flex-item2"><h2>Multi-Post Stories Gain+Glory</h2></div>
+                <div class="flex-item3">
+                    <ul class="inner-flex-item">
+                        <li class="pill-btn">Ruby on rails</li>
+                        <li class="pill-btn">css</li>
+                        <li class="pill-btn">javaScript</li>
+                        <li class="pill-btn">html</li>
+                    </ul>
+                </div>
+                <div class="flex-item4"><button>See Project</button></div>
+            </div>
```

Comment:
- [ ] Please make sure the button (e.g., "See project") is a link `a` or `button` tag, and follow the button styling: Active, Hover and Pressed ([see design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122))🤔

Reply:
working on this now

Reply:
Done